circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  node:
+    version: stable
+
+dependencies:
+  pre:
+    - google-chrome --version
+    - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+    - for v in $(curl http://archive.ubuntu.com/ubuntu/pool/main/n/nss/ | grep "href=" | grep "libnss3.*deb\"" -o | grep -o "libnss3.*deb" | grep "3.28" | grep "14.04"); do curl -L -o $v http://archive.ubuntu.com/ubuntu/pool/main/n/nss/$v; done && rm libnss3-tools*_i386.deb libnss3-dev*_i386.deb
+    - sudo dpkg -i google-chrome.deb || true
+    - sudo dpkg -i libnss3*.deb || true
+    - sudo apt-get update
+    - sudo apt-get install -f || true
+    - sudo dpkg -i libnss3*.deb
+    - sudo apt-get install -f
+    - sudo apt-get install --only-upgrade lsb-base
+    - sudo dpkg -i google-chrome.deb
```

Comment:
This whole thing looks like a lot of fun, there is no simpler way? I seem to remember that one `apt-get install -f` should be enough after trying to install the .deb, then try to install the .deb again. 

Or, possibly try using `gdebi` to figure out and install dependencies.

Reply:
I think that file is pretty old, I'll grab a more recent one from some related project.